lib/pure/collections/sequtils.nim

```diff
@@ -426,21 +426,22 @@ proc apply*[T](s: var openArray[T], op: proc (x: T): T {.closure.})
   for i in 0 ..< s.len: s[i] = op(s[i])
 
 proc apply*[T](s: openArray[T], op: proc (x: T) {.closure.}) {.inline, since: (1, 3).} =
-  ## Same as `apply` but for proc that do not return and do not mutate `s` directly.
+  ## Same as `apply` but for a proc that does not return anything
+  ## and does not mutate `s` directly.
   runnableExamples:
     var message: string
     apply([0, 1, 2, 3, 4], proc(item: int) = message.addInt item)
     assert message == "01234"
   for i in 0 ..< s.len: op(s[i])
 
 iterator filter*[T](s: openArray[T], pred: proc(x: T): bool {.closure.}): T =
-  ## Iterates through a container `s` and yields every item that fulfills the
-  ## predicate `pred` (function that returns a `bool`).
+  ## Iterates through a container `s` and yields every item that fulfilled the
```

Comment:
always prefer present tense

Reply:
Ok, I used past tense here since the documentation for the `filter` and `keepIf` procs and the `filterIt`, `keepItIf` and `countIt` templates also use the past tense. I will change that.

Reply:
I was gonna suggest adding this rule to contributing.rst to avoid back and forths in future but turns out `present tense` is already there:

> The documentation may be limited to one sentence fragment, but if multiple sentences are within the documentation, each sentence after the first should be complete and in present tense.


so, all good